backend/inspirehep/editor/views.py

```diff
@@ -86,3 +91,76 @@ def get_revision(transaction_id, rec_uuid):
         return jsonify(revision.json)
     except Exception:
         raise EditorGetRevisionError
+
+
+@blueprint.route("/<endpoint>/<int:pid_value>/rt/tickets/create", methods=["POST"])
+@login_required_with_roles([Roles.cataloger.value])
+def create_rt_ticket(endpoint, pid_value):
+    """View to create an rt ticket"""
+    json = request.json
+    ticket_id = tickets.create_ticket(
+        json["queue"],
+        current_user.email,
+        json.get("description"),
+        json.get("subject"),
+        pid_value,
+        Owner=json.get("owner"),
+    )
+    if ticket_id != -1:
+        return jsonify(
+            success=True,
+            data={
+                "id": str(ticket_id),
+                "link": tickets.get_rt_link_for_ticket(ticket_id),
+            },
+        )
+    else:
+        return jsonify(success=False), 500
+
+
+@blueprint.route(
+    "/<endpoint>/<pid_value>/rt/tickets/<ticket_id>/resolve", methods=["GET"]
+)
+@login_required_with_roles([Roles.cataloger.value])
+def resolve_rt_ticket(endpoint, pid_value, ticket_id):
+    """View to resolve an rt ticket"""
+    tickets.resolve_ticket(ticket_id)
+    return jsonify(success=True)
+
+
+@blueprint.route("/<endpoint>/<pid_value>/rt/tickets", methods=["GET"])
+@login_required_with_roles([Roles.cataloger.value])
+def get_tickets_for_record(endpoint, pid_value):
+    """View to get rt ticket belongs to given record"""
+    tickets_for_record = tickets.get_tickets_by_recid(pid_value)
+    simplified_tickets = [
+        _simplify_ticket_response(ticket) for ticket in tickets_for_record
+    ]
+    return jsonify(simplified_tickets)
+
+
+@blueprint.route("/rt/users", methods=["GET"])
+@editor_use_api_permission.require(http_exception=403)
+def get_rt_users():
+    """View to get all rt users"""
+
+    return jsonify(tickets.get_users())
+
+
+@blueprint.route("/rt/queues", methods=["GET"])
+@editor_use_api_permission.require(http_exception=403)
```

Comment:
shouldn't everything just be `@login_required_with_roles([Roles.cataloger.value])` ?

Reply:
In next there were two different kinds of required permissions:
```@editor_permission``` and ```@editor_use_api_permission```. The first one corresponds to  ```@login_required_with_roles``` whereas the second is set up here 
https://github.com/inspirehep/inspirehep/pull/1213/files#diff-0d4f912028d824fde064f50b4008099dR51-R55
To me it looks that it can be replaced with ```@login_required_with_roles``` but I wasn't sure. If that's true, can you confirm please?

Reply:
I don't know why it is like that, but to me it is OK to just require the cataloger role, AFAIK this endpoints are used only for editor, which requires cataloger role anyway.